appium/webdriver/common/appiumby.py

```diff
@@ -32,3 +34,19 @@ class AppiumBy(By):
     FLUTTER_INTEGRATION_KEY = '-flutter key'
     FLUTTER_INTEGRATION_TEXT = '-flutter text'
     FLUTTER_INTEGRATION_TEXT_CONTAINING = '-flutter text containing'
+
+ByType = Literal[
```

Comment:
where this type is supposed to be used?

Reply:
In this package is not used since some the function is derive from selenium.

This will be helpful in other packages which uses Appium, since Selenium provides the `By` types to annotate 
https://github.com/SeleniumHQ/selenium/blob/17a2c339ea4bb8ab2d57f7e60633974ed07aaa94/py/selenium/webdriver/common/by.py#L51 can Union the types easily.

Reply:
would it be possible to use constants here instead of hardcoding string values, e.g. AppiumBy.IOS_PREDICATE  instead of '-ios predicate string'?

Reply:
Python doesn't allow to a variable in Type Literal

example
![Screenshot_20241129_102440](https://github.com/user-attachments/assets/8c80c46b-553c-41c1-975b-f18ec129b093)